### PR TITLE
feat(datastore): expose datastore Close()

### PR DESCRIPTION
**Pull request by ywh254**
ref #97 Datastores should expose Close()

### CODE REVIEW

**Review by Stebalien — APPROVED** [review]
All datastores appear to now implement `Close` so this should be good to go. However, @magik6k should sign off on it.

File: autobatch/autobatch.go [hunk 1]
```diff
@@ -112,3 +112,7 @@ func (d *Datastore) Query(q dsq.Query) (dsq.Results, error) {
 func (d *Datastore) DiskUsage() (uint64, error) {
 	return ds.DiskUsage(d.child)
 }
+
+func (d *Datastore) Close() error {
```
Comment (bigs) [inline autobatch/autobatch.go:116]: should this not propagate?

Comment (magik6k) [reply autobatch/autobatch.go:116]: Yep, this should call `d.child.Close()`

Comment (ywh254) [reply autobatch/autobatch.go:116]: yes，i changed and push again

**Review by magik6k — APPROVED** [review]

**Pull request merged**